test-results/failing/GFM/[extension] Tables.md

```diff
@@ -84,71 +20,5 @@ Should give output:
 But instead was:
 
 ````````````html
-<p>| abc | def | | --- | --- | | bar | baz |</p><blockquote><p>bar</p></blockquote>
-````````````
-## [Example 202](https://github.github.com/gfm/#example-202)
-
-This markdown:
-
-````````````markdown
-| abc | def |
-| --- | --- |
-| bar | baz |
-bar
-
-bar
-````````````
-
-Should give output:
-
-````````````html
-<table><thead><tr><th>abc</th><th>def</th></tr></thead><tbody><tr><td>bar</td><td>baz</td></tr><tr><td>bar</td><td></td></tr></tbody></table><p>bar</p>
-````````````
-
-But instead was:
-
-````````````html
-<p>| abc | def | | --- | --- | | bar | baz | bar</p><p>bar</p>
-````````````
-## [Example 204](https://github.github.com/gfm/#example-204)
-
-This markdown:
-
-````````````markdown
-| abc | def |
-| --- | --- |
-| bar |
-| bar | baz | boo |
-````````````
-
-Should give output:
-
-````````````html
-<table><thead><tr><th>abc</th><th>def</th></tr></thead><tbody><tr><td>bar</td><td></td></tr><tr><td>bar</td><td>baz</td></tr></tbody></table>
-````````````
-
-But instead was:
-
-````````````html
-<p>| abc | def | | --- | --- | | bar | | bar | baz | boo |</p>
-````````````
-## [Example 205](https://github.github.com/gfm/#example-205)
-
-This markdown:
-
-````````````markdown
-| abc | def |
-| --- | --- |
-````````````
-
-Should give output:
-
-````````````html
-<table><thead><tr><th>abc</th><th>def</th></tr></thead></table>
-````````````
-
-But instead was:
-
-````````````html
-<p>| abc | def | | --- | --- |</p>
+<table><thead><tr><th>abc</th><th>def</th></tr></thead><tbody><tr><td>bar</td><td>baz</td></tr><tr><td>&gt; bar</td><td></td></tr></tbody></table>
```

Comment:
I see that this is still here even though it's listed as passing in test-results/passing-GFM. Is it possible that when there are no failing tests in a file it doesn't update that file to be empty?